metadata/src/test/java/org/apache/kafka/metadata/KafkaConfigSchemaTest.java

```diff
@@ -61,9 +62,9 @@ public class KafkaConfigSchemaTest {
     public static final Map<String, List<ConfigSynonym>> SYNONYMS = new HashMap<>();
 
     static {
-        SYNONYMS.put("abc", Arrays.asList(new ConfigSynonym("foo.bar")));
-        SYNONYMS.put("def", Arrays.asList(new ConfigSynonym("quux", HOURS_TO_MILLISECONDS)));
-        SYNONYMS.put("ghi", Arrays.asList(new ConfigSynonym("ghi")));
+        SYNONYMS.put("abc", Collections.singletonList(new ConfigSynonym("foo.bar")));
```

Comment:
I wonder if it's more readable to actually keep `Arrays.asList()` here so all lines are similar